### ISSUE TITLE
Content in the dataset catalog

### ISSUE THREAD

**Issue by Chatoyant19** [bittnt/Objectness#17]
Hi,I have a question about the VOC 2007 dataset.Can you tell me whether the training set and test set should be included separately under the VOC 2007 folder, or Annotations and ImageSets and etc. these folder are directly under VOC 2007 dataset?
